### PR TITLE
Add editor config.

### PR DESCRIPTION
**Pull request by rcorral**
Updated Contributing guidelines and reame.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by daiyiwhale** [comment]
:ship:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,9 +82,11 @@ Before you submit your pull request consider the following guidelines:
      git checkout -b my-fix-branch master
      ```
 
+* Setup your editor http://editorconfig.org/#download
```
Comment (kennyt) [inline CONTRIBUTING.md]: maybe we can do:
`[Setup your editor](http://editorconfig.org/#download)`

instead of just having the link

**Comment by kennyt** [comment]
:ship: after one comment

**Pull request merged**